CHANGELOG.md

```diff
@@ -2,7 +2,6 @@
 
 ## Not released
 - Fix PieWidget tooltip confining it inside canvas [#130](https://github.com/CartoDB/carto-react/pull/130)
-
 - Fix FormulaWidgetUI render when data is zero [#128](https://github.com/CartoDB/carto-react/pull/128)
```

Comment:
I would add this explictly to the CHANGELOG, as we're removing a public prop: 

Remove viewportFilter property from widgets [#31](https://github.com/CartoDB/carto-react/pull/131/)

Reply:
I'll do it myself, don't worry about it